libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -342,6 +344,16 @@ dictionary LspInformation {
     OpeningFeeParamsMenu opening_fee_params_list;
 };
 
+dictionary OpenChannelFeeRequest {
+    u64 amount_msat;
```

Comment:
Most requests made to the SDK are in sats, especially the ReceivePaymentRequest. With this we are asking the developer to convert sats/msats for calls to open_channel_fee and receive_payment.

Reply:
Might make sense to switch to msats everywhere, since this is the native unit on LN.

I opened #417 to track this for a future release.

Reply:
@dangeross are you OK with that?